### PR TITLE
Fix/sni 6056 set background removal parameter

### PR DESCRIPTION
**Pull request by AlexPinhasov**
#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [x] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

**Comment by const-cloudinary** [comment]
@AlexPinhasov , please add unit tests.

**Comment by AlexPinhasov** [comment]
@const-cloudinary Unit test added.

File: Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift [hunk 1]
```diff
@@ -884,6 +884,39 @@ class UploaderTests: NetworkBaseTest {
...
+        XCTAssertNotNil(cloudinary!.config.apiSecret, "Must set api secret for this test")
+
+        let expectation = self.expectation(description: "Upload should succeed")
+        let resource: TestResourceType = .borderCollie
+        let file = resource.url
+        var result: CLDUploadResult?
+        var error: NSError?
+
+        let params = CLDUploadRequestParams()
+        params.setBackgroundRemoval("cloudinary_ai")
```
Comment (const-cloudinary) [inline Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift:899]: Please see:
https://github.com/cloudinary/pycloudinary/blob/73d84511eb18d2f4198d6375059a5d94716fc0c4/test/test_uploader.py#L781

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift (hunk 1 above)
Comment (AlexPinhasov) [reply Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift]: Done

**Pull request merged**